Firebase/InstanceID/FIRInstanceID+Private.h

```diff
@@ -52,4 +52,12 @@
  */
 - (nullable NSString *)appInstanceID:(NSError *_Nullable *_Nullable)error;
 
+/**
+ *  Returns a Firebase Messaging scoped token for the firebase app.
+ *
+ *  @return Returns the stored token if the device has registered with Firebase Messaging, otherwise
+ *          returns nil.
+ */
+- (nullable NSString *)token;
```

Comment:
This is failing to build because FIRInstanceID+Private.h is really an internal header, not a private header.

I recommend renaming this header to  FIRInstanceIDInternal.h and creating a new FIRInstanceIDPrivate.h in a Private directory that is referenced from the podspec like https://github.com/firebase/firebase-ios-sdk/blob/master/FirebaseCore.podspec#L29.

That should get the build working. To get the travis pod lib lint tests, there will need to be a preliminary version of FirebaseInstanceID.podspec at https://github.com/firebase/SpecsStaging. I can help with that once the build is fixed.